src/lC_common_h.jl

```diff
@@ -1,705 +1,1253 @@
-macro c(ret_type, func, arg_types, lib)
-  local args_in = [Symbol('a', i) for i in eachindex(arg_types.args)]
-  quote
-    $(esc(func))($(args_in...)) = ccall( ($(string(func)), $lib ), $ret_type, $arg_types, $(args_in...) )
-  end
-end
+# Automatically generated using Clang.jl wrap_c, version 0.0.0
 
-macro ctypedef(fake_t,real_t)
-  quote
-    const $(esc(fake_t)) = $(esc(real_t))
-  end
-end
+using Compat
+
+const CURL_SOCKET_BAD = -1
+
+# begin enum ANONYMOUS_1
+const ANONYMOUS_1 = UInt32
```

Comment:
Not sure why these ANONYMOUS constants were generated or if they are needed.

Reply:
I think they come from badly parsed enums. I am currently going through them all to update them.